tests/lib/location/prj.conf

```diff
@@ -30,3 +30,5 @@ CONFIG_MOCK_NRF_MODEM_AT=y
 CONFIG_LOG=n
 CONFIG_LOCATION_LOG_LEVEL_DBG=n
 CONFIG_LTE_LINK_CONTROL_LOG_LEVEL_DBG=n
+
+CONFIG_LOCATION_METHOD_WIFI_NET_MGMT=n
```

Comment:
You cannot set this Kconfig. It's not user configurable:
```
error: LOCATION_METHOD_WIFI_NET_MGMT (defined at /jenkins_workspace/workspace/latest_sdk-
nrf_PR-24297/nrf/lib/location/Kconfig:36) is assigned in a configuration file, but is not directly
user-configurable (has no prompt). It gets its value indirectly from other symbols.
```

Reply:
alright, I will remove that change

Reply:
This is still there.